addons/website/static/tests/tours/popup_visibility_option.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module */
+
+import wTourUtils from 'website.tour_utils';
+
+wTourUtils.registerWebsitePreviewTour('website_popup_visibility_option', {
+    test: true,
+    edition: true,
+    url: '/',
+}, [
+    wTourUtils.dragNDrop({
+        id: "s_popup",
+        name: "Popup",
+    }),
+    {
+        content: "Click on the column within the popup snippet.",
+        trigger: "iframe #wrap .s_popup .o_cc1",
+    },
+    {
+        content: "Click the 'No Desktop' visibility option.",
+        trigger: ".snippet-option-DeviceVisibility we-button[data-toggle-device-visibility='no_desktop']",
+    },
+    {
+        content: "Verify that the popup is visible and the column is invisible.",
+        trigger: ".o_we_invisible_root_parent",
+        run: () => {
+            const isPopupVisible = document.querySelector(".o_we_invisible_root_parent i").classList.contains("fa-eye");
+            const isColumnInvisible = document.querySelector("ul .o_we_invisible_entry i").classList.contains("fa-eye-slash");
+            if (!isPopupVisible || !isColumnInvisible) {
+                console.error("Visibility issue detected with the elements.");
+            }
```

Comment:
@paru-odoo this will do : 

```suggestion
   {
        content: "Verify that the popup is visible and the column is invisible.",
        trigger: ".o_we_invisible_root_parent i.fa-eye, ul .o_we_invisible_entry i.fa-eye-slash",
    },
```